File: Sec6-WhileLoops/SafeSquareRoot.java
import java.util.Scanner;
public class SafeSquareRoot {
  public static void main(String[] args) {

    Scanner keyboard = new Scanner(System.in);
    int number;

    System.out.println("SQUARE ROOT!");
    System.out.print("Enter a number: ");
    number = keyboard.nextInt();

    while(number < 0){
      System.out.println("You can't take the square root of a negative number, silly.");
      System.out.print("Try again: ");
      number = keyboard.nextInt();
    }
    System.out.println("The square root of " + number + " is " + Math.sqrt(number) +".");
  }

}

File: Sec6-WhileLoops/Safe_Square_Root.py
import math

print("SQUARE ROOT!")
input_number = input("Enter a number: ")
number = int(input_number)

while number < 0:
    print("You can't take the square root of a negative number, silly.")
    number = int(input("Try again: "))

print("The square root of " + str(number) + " is " + str(math.sqrt(number)) +".")
